README.md

```diff
@@ -2,22 +2,69 @@
 
 [![JB Research](https://jb.gg/badges/research-flat-square.svg)](https://research.jetbrains.org/)
 
-KInference is a library that makes possible execution of complex ML models (written via ONNX) in vanilla Kotlin.
+KInference is a library that makes possible execution of complex ML models (written via ONNX) in Kotlin.
 
 [ONNX](https://github.com/onnx/onnx) is a popular ecosystem for building, training, evaluating, and exchanging ML and DL models. It makes the process much
 simpler and divides the model into building blocks that can be switched or tuned to one's liking.
 
-However, ONNX carries with itself a lot of dependencies and requirements that complicate its use in some cases. Our library does not require anything but
-vanilla Kotlin. KInference is lightweight but fast, and supports numerous ONNX operators. This makes it easier to use and is especially useful for various
-applications that require the models to be run on the users' machines.
+However, ONNX carries with itself a lot of dependencies and requirements that complicate its use in some cases. 
+KInference is designed to facilitate ONNX models inference on a variety of platforms via configurable backends.
+Our library addresses the problem of local inference as well and provides several solutions that are suitable for running on users' machines. 
 
 Right now, KInference is in active development.
 
-## Setup
+### Table of contents:
+* [Why should I use KInference?](#why-should-i-use-kinference)
+* [KInference Backends](#kinference-backends)
+* [Getting started](#getting-started)
+* [Examples](#examples)
 
+### Why should I use KInference?
```

Comment:
I think we must add here that KInference has pure-JS and pure-JVM backends that make possible running models anywhere where JS or JVM virtual machine is available.